lib/assessment/label.py

```diff
@@ -104,20 +109,41 @@ def bedrock_anthropic_label_student_work(self, prompt, rubric, student_code, stu
             raise Exception(f"Error parsing llm_model: {llm_model} bedrock_model: {bedrock_model}")
 
         anthropic_prompt = self.compute_anthropic_prompt(prompt, rubric, student_code, examples=examples)
-        body = json.dumps({
-            "prompt": anthropic_prompt,
-            "max_tokens_to_sample": 4000,
-            "temperature": temperature,
-            # "top_p": 0.9,
-        })
+        if "claude-3" in bedrock_model:
+            body = json.dumps({"anthropic_version": "bedrock-2023-05-31",
+                               "max_tokens": 4000,
```

Comment:
It was just a guess. What would you recommend as a max value?

Reply:
TLDR: I vote for removing this line.

upon further inspection, it doesn't look like `max_tokens` is in the request API: https://docs.anthropic.com/en/api/complete there is a `max_tokens_to_sample`. 

according to https://docs.anthropic.com/en/docs/about-claude/models, the default limit is 4096 tokens, which can be increased to 8192 tokens by using a special header. 

so, let's just use the default limit, and if we notice that we are getting a lot of responses rejected due to hitting token limits before 8192 becomes the default, we can consider using the header to increase the limit.

I did check the reports in aitt_release_data for the latest release, and I didn't see any errors there, which is encouraging.

Reply:
Turns out max_tokens is required. Messages to Claude 3 sent without it fail.
`botocore.errorfactory.ValidationException: An error occurred (ValidationException) when calling the InvokeModel operation: Malformed input request: #: subject must not be valid against schema 
{
    "required": [
        "messages"
    ]
}
#: required key [max_tokens] not found, please reformat your input and try again.`

Reply:
I'm adding it back in and setting it to the default 4096 value. If we get errors, I'll bump it up to 8192.

Reply:
sounds good. for full disclosure, my intuition on this may have been off because I was thinking of the limits we ran into with context window (including both inputs and outputs). I don't believe we've run into any problems with the limit of 4096 output tokens. most other models we've tried have had the same limit on output tokens.